src/generators/src/infra/database/migrations/migrations.js

```diff
@@ -47,11 +44,35 @@ module.exports =
           const columns = []
           Object.keys(schema).forEach((prop) => {
             const { name, type, options } = schema[prop]
-            columns.push(`table.${type2Str(type)}('${snakeCase(name)}')${options.isId ? '.primary()' : ''}`)
+
+            const nativeTypes = [Boolean, Number, String, Date]
+            if (!nativeTypes.includes(type)) return
+
+            const typeString = type2Str(type)
+            columns.push(`table.${typeString}('${snakeCase(name)}')${options.isId ? '.primary()' : ''}`)
           })
           return columns
         }
 
+        function isArrayWithType(value) {
+          return Array.isArray(value) && value.length === 1
+        }
+
+        function identifyRef(schema) {
+          const refs = []
+          Object.values(schema).forEach(({ type, name }) => {
+            if (herbs.entity.isEntity(type)) {
+              const typeSchema = type.prototype.meta.schema
+              const idRef = Object.values(typeSchema).find(column => column.options.isId)?.name
+              refs.push({ id: idRef, columnName: `${camelCase(name)}Id`, table: `${camelCase(type.name)}s`, relationship: 'One-to-One' })
+            }
+
+            if (isArrayWithType(type) && herbs.entity.isEntity(type[0]))
+              refs.push({ table: `${camelCase(type[0].name)}s`, relationship: 'One-to-Many' })
```

Comment:
Both table and columns name should be snake_cases for DB to keep with the current standard. 

ex: `userId` -> `user_id`